config/jobs/periodic/cluster-api-provider-ibmcloud/test-e2e-capi-ibmcloud-periodics.yaml

```diff
@@ -28,3 +28,33 @@ periodics:
             - "./scripts/ci-e2e.sh"
           securityContext:
             privileged: true
+  - name: periodic-capi-provider-ibmcoud-e2e-vpc
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    decorate: true
+    decoration_config:
+      gcs_configuration:
+        bucket: ppc64le-kubernetes
+        path_strategy: explicit
+      gcs_credentials_secret: gcs-credentials
+    interval: 3h
+    extra_refs:
+      - base_ref: main
+        org: kubernetes-sigs
+        repo: cluster-api-provider-ibmcloud
+        workdir: true
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220916-c3af09ab20-1.24
+          env:
+            - name: "E2E_FLAVOR"
+              value: "vpc"
+          command:
+            - "runner.sh"
+            - "./scripts/ci-e2e.sh"
+          envFrom:
+            - secretRef:
+                name: ibm-cloud-vpc-credentials
```

Comment:
what's this credential used for? we are suppose to use the secret from the boskos itself right?

Reply:
@mkumatag, In this job, we haven't integrated Boskos yet and hence this credential was used which would have the API key unlike the PowerVS flow which fetches the API key(part of user data) on acquiring the boskos resource.